.cirrus.yml

```diff
@@ -0,0 +1,53 @@
+env:
+  MAVEN_OPTS: -Xmx512M -XX:+ExitOnOutOfMemoryError
+  MAVEN_SKIP_CHECKS_AND_DOCS: -Dair.check.skip-all=true -Dmaven.javadoc.skip=true
+  MAVEN_FAST_INSTALL: -DskipTests $MAVEN_SKIP_CHECKS_AND_DOCS -B -q -T 1C
+
+webui_checks_task:
+  container:
+    image: node:latest
+  check_script: presto-main/bin/check_webui.sh
+
+maven_checks_task:
+  container:
+    image: maven:3-jdk-8
+  maven_cache:
+    folder: ~/.m2
+  test_script: mvn install -DskipTests -B -T 1C -P ci
+  docker_script: docker/build-local.sh
+
+maven_specific_modules_task:
+  container:
+    image: maven:3-jdk-8
+  maven_cache:
+    folder: ~/.m2
+  env:
+    matrix:
+      - TEST_MODULES: presto-main
+      - TEST_MODULES: presto-tests
+      - TEST_MODULES: presto-raptor-legacy
+      - TEST_MODULES: presto-accumulo
+      - TEST_MODULES: presto-cassandra
+      - TEST_MODULES: presto-hive,presto-orc,presto-parquet
+      - TEST_MODULES: presto-mongodb,presto-kafka,presto-elasticsearch
+      - TEST_MODULES: presto-redis
+      - TEST_MODULES: presto-sqlserver,presto-postgresql,presto-mysql
+      - TEST_MODULES: presto-phoenix,presto-iceberg
+  install_script: mvn install $MAVEN_FAST_INSTALL -pl "${TEST_MODULES}" -am
+  test_script: mvn test $MAVEN_SKIP_CHECKS_AND_DOCS -B -pl "${TEST_MODULES}" $TEST_FLAGS
+
+maven_presto_tests_ci_task:
+  container:
+    image: maven:3-jdk-8
+  maven_cache:
+    folder: ~/.m2
+  install_script: mvn install $MAVEN_FAST_INSTALL -pl presto-tests -am
+  test_script: mvn test $MAVEN_SKIP_CHECKS_AND_DOCS -B -pl presto-tests -P ci-only
+
+maven_other_modules_task:
```

Comment:
I don't see any dockerized builds here. How are they handled?

Reply:
I haven't got that far yet. When I emailed them previously, they said to look at

https://cirrus-ci.org/guide/writing-tasks/#additional-containers
https://cirrus-ci.org/guide/docker-builder-vm/